api/CSSKeyframesRule.json

```diff
@@ -0,0 +1,346 @@
+{
+  "api": {
+    "CSSKeyframesRule": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/CSSKeyframesRule",
+        "support": {
+          "chrome": {
+            "version_added": true
+          },
+          "chrome_android": {
+            "version_added": true
+          },
+          "edge": {
+            "version_added": true
+          },
+          "edge_mobile": {
+            "version_added": true
+          },
+          "firefox": [
+            {
+              "version_added": "48"
+            },
+            {
+              "version_added": "5",
+              "prefix": "moz"
+            }
+          ],
+          "firefox_android": [
+            {
+              "version_added": "48"
+            },
+            {
+              "version_added": "5",
+              "prefix": "moz"
+            }
+          ],
+          "ie": {
+            "version_added": "10"
+          },
+          "opera": {
+            "version_added": "12",
+            "prefix": "-o"
+          },
+          "opera_android": {
+            "version_added": "12",
+            "prefix": "-o"
+          },
+          "safari": {
+            "version_added": "4"
+          },
+          "safari_ios": {
+            "version_added": true
+          },
+          "webview_android": {
+            "version_added": true
+          }
+        },
+        "status": {
+          "experimental": true,
+          "standard_track": false,
```

Comment:
This should be standard (subfeatures as well) https://drafts.csswg.org/css-animations/#interface-csskeyframesrule

cc: @ddbeck

Reply:
Opened #2502